src/main/java/de/embl/cba/mobie/n5/zarr/N5OMEZarrImageLoader.java

```diff
@@ -244,6 +244,7 @@ private Multiscale getMultiscale( String pathName ) throws IOException
 			if ( n5 instanceof N5S3ZarrReader )
 			{
 				final N5S3ZarrReader s3ZarrReader = ( N5S3ZarrReader ) n5;
+				s3ZarrReader.setDimensionSeparator("/");
```

Comment:
I don't really think this change is necessary, as all the changes here are only done to generate the message for the exception thrown below.